website/forms/php/global-variables/geolocation.md

```diff
@@ -3,4 +3,66 @@ id: geolocation
 title: Geolocation
 ---
 
-*Coming soon*
+### ES_GEOLOCATION_USE
+
+This variable will force geolocation to always be in use, regardless of the value set in Settings.
+
+```php
+define('ES_GEOLOCATION_USE', '<true|false>');
+```
+
+### ES_GEOLOCATION_IP
+
+Forces the geolocation system to use the provided IP address. Can be useful for testing.
```

Comment:
```suggestion
Setting this variable will force the geolocation system to use the provided IP address instead of the user's actual one. Can be useful for testing, especially in local setups where `127.0.0.1` / `localhost` will be ignored.
```